topics/RDMA.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
```

Comment:
experimental support

We should add a note that it can break at any minor version, lole we wrote in the README in the code repo.

Reply:
> experimental support
> 
OK.

> We should add a note that it can break at any minor version, like we wrote in the README in the code repo.

Please take a look at `Limitations` section, there is a note `Valkey Over RDMA is experimental, it may be changed or be removed in any minor or major version.`.
Should I describe it at other place?

Reply:
OK, I didn't see this before. Limitations section is good.

Reply:
> OK, I didn't see this before. Limitations section is good.

Hope this doesn't let you feel pushy...
I guess we need merge it for Valkey 8.0.

Reply:
We don't have versioned docs. They are mainly online. The docs are not included in any release, though distros may generate man pages from them when packaging Valkey 8. We can merge it just before Valkey 8. Thus, no panic.